Library/Homebrew/formula.rb

```diff
@@ -1063,6 +1063,18 @@ def std_cmake_args
     ]
   end
 
+  # Returns the path to a make 3.81 or newer binary.
+  # Returns the system make on Leopard and newer, and the
+  # path to the actually-installed make on Tiger or older.
+  def make_path
```

Comment:
While, may I ask could we make it use superenv? How hard is it? Because mixing code which may should belong to superenv in formula class feels a bit code smell to me.